assets/js/voronoi.js

```diff
@@ -113,8 +124,10 @@ function voronoi() {
 
   ctx.putImageData(imageData, 0, 0);
   points.forEach((point) => {
-    ctx.fillStyle = "black";
-    ctx.fillRect(point.x, point.y, 2, 2);
+    if (showDotsCheckbox?.checked) {
```

Comment:
Wow I didn't know that this operator is now available in browsers, thanks :).
By the way. Move this if line up. We don't have to do this loop at all.